build_early.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+BRANCH=early
+
+cd `dirname $0`/..
+. site/build_vars.sh
```

Comment:
Can you explain this part? This moves to the parent directory and tries to source the `build_vars.sh`. But this will fail if I checked out the code under another directory name than `site`. The following `git commands` will also fail, because I am now in the parent dir, which is not a git repository.

Reply:
I will add a check that the current directory is names site and exit otherwise

Reply:
I suggest to use a variable for the current directory name:
```bash
SITE_DIR=$(basename "$(pwd)")
```

Why not build directly in the site repro directory?

Reply:
My fault. I forgot that it is a subdirectory in the gluon repository.